lib/ews/soap/ews_soap_get_user_settings_response.rb

```diff
@@ -0,0 +1,50 @@
+=begin
+  This file is a cotribution to Viewpoint; the Ruby library for Microsoft Exchange Web Services.
+
+  Copyright © 2013 Mark McCahill <mark.mccahill@duke.edu>
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+=end
+
+module Viewpoint::EWS::SOAP
+  class EwsSoapGetUserSettingsResponse < EwsSoapResponse
+    include Viewpoint::StringUtils
```

Comment:
We might need to add some more methods to this once we can test with real data, will see then.

Reply:
some methods already in `EwsSoapResponse` so we can just reuse it - but for now it's not big deal as we don't know what we will get back from EWS so can leave it as now

Reply:
Yeah tbh I wasn't even sure if we need this class right now, we could just use `EwsSoapResponse`. But I was thinking that once we get some real data from the response, we can have methods that get you the relevant data, like for example the `settings` that are returned.